swagger.yaml

```diff
@@ -10,11 +10,11 @@ tags:
     description: Aka Politician
 
 paths:
-  /organizations/{id}:
+  /organizations/{id}?start_date={start_date}&end_date={end_date}:
```

Comment:
Follow the Query Parameters section of the [OpenAPI Specifications](https://swagger.io/docs/specification/describing-parameters/#query-parameters). You put query paramters in the `parameters` section so they show up as inputs, like `additionalMetadata` and `file` in this example

![image](https://user-images.githubusercontent.com/17074399/160252050-4b32b677-0895-4a70-8ced-df2dde84a779.png)
https://petstore.swagger.io/